packages/dev-util/idea-ultimate/files/idea-ultimate.desktop

```diff
@@ -0,0 +1,12 @@
+# Based on autogenerated by IDEA 2020 desktop entry
```

Comment:
So it’s generated somewhere?

Reply:
![image](https://user-images.githubusercontent.com/7518245/232905112-5d24c2bc-1f03-42b1-b13c-f7fa2d2c29ee.png)
IntelliJ itself has a tool to create it. I just copied it and adapted it (a bit).